src/components/ExceededCommentLength.js

```diff
@@ -15,7 +15,7 @@ const ExceededCommentLength = (props) => {
     }
 
     return (
-        <Text style={[styles.textMicro, styles.textDanger, styles.chatItemComposeSecondaryRow, styles.mlAuto]}>
+        <Text style={[styles.textMicro, styles.textDanger, styles.chatItemComposeSecondaryRow, styles.mlAuto, styles.pl1]}>
```

Comment:
Let's use `pl2` because the space is too little right now (it's not available right now, you need to add it [here](https://github.com/Expensify/App/blob/093634cf7bc3497e826e62b54c7a1f7f3207a2f4/src/styles/utilities/spacing.js#L308))